### ISSUE TITLE
The target "GetBuildVersion" does not exist in the project

### ISSUE THREAD

**Issue by v-cagome** [libgit2/libgit2sharp#1503]
Just cloned and tried to build in VS 2017 and it fails with error: **The target "GetBuildVersion" does not exist in the project**

Any ideas?

**Comment by ethomson** [comment]
Right.  VS 2017 and msbuild seem to not be playing together very nicely sometimes.  Can you try explicitly restoring the nuget packages from within VS 2017?

If that doesn't work, try restoring from the command line: `msbuild /t:restore` or `dotnet restore`.  (I do this with VS 2017 closed, though I'm not sure if this is actually necessary.)
